recipe/run_client_tests.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash -ex
+
+# Parts of the test code are based on output generated by AI
+
+# Function to cleanup MySQL server
+cleanup() {
+    echo "Shutting down MySQL server..."
+    mysqladmin -u root shutdown
+    rm -rf /tmp/mysqlclienttest
+}
+
+# Set trap to call cleanup function on script exit
+trap 'mysqladmin -u root shutdown || true; rm -rf /tmp/mysqlclienttest' EXIT
+
+# Initialize
+mkdir -p /tmp/mysqlclienttest/data
+mysqld --initialize-insecure --datadir=/tmp/mysqlclienttest/data
+
+# Create error log
+touch ./data/error.log
+
+# Start server
+mysqld --user=root --datadir=/tmp/mysqlclienttest/data --pid-file=/tmp/mysqlclienttest/mysql.pid --port=33071 &
+
+sleep 3
+
+# Create database for testing
+mysql -u root -e "CREATE DATABASE test;"
+
+# Run client tests:
+#   Create client configuration
+mv mysql_test_db.cnf tests/mysql_test_db.cnf
+#   Point tests to the configuration file
+export TESTDB=mysql_test_db.cnf
+#   Run the tests
+pytest -vv tests
```

Comment:
```suggestion
TESTDB=mysql_test_db.cnf pytest -vv tests
```
I'd try that, to not pollute the environment (instead of `export`)

Reply:
Unfortunately that option (along with few other i tried) didn't work on CI. 
edit: my above comment no longer relevant, found the issue on CI was due to different cause. 
Updated code to as suggested!